.github/workflows/openapi_lint.yaml

```diff
@@ -0,0 +1,25 @@
+name: Validate OpenAPI schema with Spectral
+on:
+  workflow_call: { }
+  pull_request:
+    branches:
+      - main
+    paths:
+      - kafka-admin/.openapi/kafka-admin-rest.yaml
```

Comment:
Try a trivial change to the OpenAPI file and let's see the new workflow run on this PR. You'll need to change `kafka-admin-api/kafka-admin/src/main/resources/META-INF/openapi.yaml` and regenerate the file here.

See https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api#updating-openapi-file

Reply:
Tried with a small description change and the workflow ran looks fine.